lib/liquid/combinators/tags/generic.ex

```diff
@@ -7,5 +7,9 @@ defmodule Liquid.Combinators.Tags.Generic do
   """
   alias Liquid.Combinators.Tag
 
+  @type t :: [
```

Comment:
`@type t` -> `@type else...` because Generic is not a type, it is a helper which can contains types (tags)

Reply:
done but i put else_tag because they don't let me use else